docs/releases.md

```diff
@@ -8,6 +8,33 @@ title: Releases
 This file contains changes and migration steps for Graphback project. 
 Please follow indidividual releases for more information.
 
+# Next
+
+### Graphback
+#### Features
```

Comment:
Correct if I'm wrong but don't we need to keep the same format as others? ie 
```
## 0.8.3 (29th August, 2019)
### Templates
#### Fixes
```

Reply:
Yes, I am also unsure about this. This new section doesn't really seem to "fit" in with the rest.

Reply:
It is not clear what will be the next version so I'm using Next to stage it and going to replace it on release.

Format is right see bellow:

```
## 0.8.0 (28th August, 2019)
### Graphback
#### Features	
```

Reply:
> ## 0.9.0 (5th September, 2019)

So I do not know the version number and date- this is going to be done on release.